nimlsp/README.rst

```diff
@@ -0,0 +1,244 @@
+==========
+Nim Language Server Protocol
```

Comment:
can you rename from `Nim` -> `Nimskull`

Reply:
I think it's too earlier to replace all the name.